src/NHSD.BuyingCatalogue.Ordering.Api/Models/ServiceRecipientModel.cs

```diff
@@ -10,10 +10,10 @@ public ServiceRecipientModel()
         {
         }
 
-        internal ServiceRecipientModel(SelectedServiceRecipient recipient)
+        internal ServiceRecipientModel(ServiceRecipient recipient)
         {
-            Name = recipient.Recipient.Name;
-            OdsCode = recipient.Recipient.OdsCode;
+            Name = recipient.Name;
+            OdsCode = recipient.OdsCode;
         }
```

Comment:
Can this constructor be removed?

Reply:
Removed constructor with PR [435](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueOrdering/pull/435)